File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 12/17/2025, 10:53:06 AM
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l = [0]*(len(nums)+1)
        for i in nums:
            l[i]=1
        for i in range(len(l)):
            if l[i]==0:
                return i   

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 12/17/2025, 10:53:02 AM
import java.util.Arrays;
class Solution {
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        int len = nums.length;
        int check = 0;
        while (check<len) {
            if (check!=nums[check]) {
                return check;
            }
            check++;
        }
        return nums.length;
    }
}